staging.hcl

```diff
@@ -0,0 +1,31 @@
+variable "branch_or_sha" {
+  type = string
+  default = "main"
+}
+job "pcdm.org-staging" {
+  region = "global"
+  datacenters = ["dc1"]
+  node_pool = "staging"
+  type = "service"
+  group "web" {
+    count = 2
+    network {
+      port "http" { to = 3000 }
+    }
+    service {
+      port = "http"
+    }
+    task "webserver" {
+      driver = "podman"
+      config {
+        image = "ghcr.io/pulibrary/pcdm.org:${ var.branch_or_sha }"
+        ports = ["http"]
+        force_pull = true
+      }
+      resources {
```

Comment:
I'd just get rid of this resources block. This says you need 1 ghz and 500 mb of memory for this static site, and I just kinda doubt that.